### PR TITLE
Downgrade to go1.21.7

### PR DESCRIPTION
**Pull request by brianstrauch**
Checklist
---------
- [x] Leave this box unchecked if features are not yet available in production

What
----
Fix a segfault that occurs in go1.22 when boringcrypto is enabled: https://github.com/golang/go/issues/65991

### CODE REVIEW

**Review by sgagniere — APPROVED** [review]
Uh oh

File: .go-version [hunk 1]
```diff
@@ -1 +1 @@
-1.22.0
+1.21.8
```
Comment (brianstrauch) [inline .go-version:1]:
```suggestion
1.21.7
```

File: go.mod [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module github.com/confluentinc/cli/v3
 
-go 1.22.0
+go 1.21.8
```
Comment (brianstrauch) [inline go.mod:3]:
```suggestion
go 1.21.7
```

File: service.yml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 name: cli
 lang: go
+lang_version: "1.21.8"
```
Comment (brianstrauch) [inline service.yml:3]:
```suggestion
lang_version: "1.21.7"
```

**Pull request merged**